package.json

```diff
@@ -151,5 +151,9 @@
     "deploy-set-version": "gulp deploy-set-version",
     "prettier": "prettier --write \"**/*\"",
     "prettier-check": "prettier --check \"**/*\""
+  },
+  "lint-staged": {
+    "*.{js,cjs,html}": ["pretty-quick", "eslint --cache --quiet"],
```

Comment:
Maybe we should run eslint before bothering to run prettier. If there's an error, we'll want to bail out early, right?